samples/Gerber Image/AGV_GPIO.gml

```diff
@@ -0,0 +1,1021 @@
+G75*
+%MOIN*%
+%OFA0B0*%
+%FSLAX25Y25*%
+%IPPOS*%
+%LPD*%
+%AMOC8*
+5,1,8,0,0,1.08239X$1,22.5*
+%
+%ADD10C,0.00000*%
+D10*
```

Comment:
Normally, we prefer larger samples over smaller ones, but a 1,000+ line file is overkill (and adds unnecessary bloat to the classifier). Any chance you could cull some of the `X0213131...` lines so the file is cut down to maybe 100-200 lines, maybe?

Reply:
Done :)